Lib/gzip.py

```diff
@@ -520,7 +520,7 @@ def _rewind(self):
         super()._rewind()
         self._new_member = True
 
-def compress(data, compresslevel=_COMPRESS_LEVEL_BEST, *, mtime=None):
+def compress(data, compresslevel=_COMPRESS_LEVEL_BEST, *, mtime=0):
     """Compress data in one shot and return the compressed string.
     Optional argument is the compression level, in range of 0-9.
     """
```

Comment:
Your fork is significantly out-of-date, this changed in 2021. Once you merge that in, the docstring should be updated.


![image](https://github.com/user-attachments/assets/fb97bf7b-6715-4c10-a7e8-81b19b5f6b9f)